src/perftest_parameters.c

```diff
@@ -34,6 +34,7 @@ static const char *portStates[] = {"Nop","Down","Init","Armed","","Active Defer"
 static const char *qp_state[] = {"OFF","ON"};
 static const char *exchange_state[] = {"Ethernet","rdma_cm"};
 static const char *atomicTypesStr[] = {"CMP_AND_SWAP","FETCH_AND_ADD"};
+static const char *congestStr[] = {"DCQCN","LDCP","HC3","DIP"};
```

Comment:
if libhns is missing then this will lead to "warning: ‘congestStr’ defined but not used "
maybe its better to add ifdef here.

Reply:
> if libhns is missing then this will lead to "warning: ‘congestStr’ defined but not used " maybe its better to add ifdef here.

Sure. Will fix this. Thanks!